Controller/SnippetAreaController.php

```diff
@@ -113,5 +127,16 @@ public function getAction(Request $request, string $area): Response
                 'Content-Type' => 'application/json',
             ]
         );
+
+        $response->setPublic();
+        $response->setMaxAge(0);
+        $response->setSharedMaxAge(0);
```

Comment:
Should be the values from: https://github.com/sulu/sulu/blob/75578fc6074db7f31d63eb4e958b1e6aac676423/src/Sulu/Bundle/HttpCacheBundle/CacheLifetime/CacheLifetimeEnhancer.php#L77-L78